x-pack/plugins/observability_solution/observability/public/routes/routes.tsx

```diff
@@ -139,44 +131,23 @@ export const routes = {
     params: {},
     exact: true,
   },
-  [SLOS_PATH]: {
+  [OLD_SLOS_PATH]: {
     handler: () => {
-      return <SlosPage />;
+      return <SimpleRedirect to="/" redirectToApp="slo" />;
     },
     params: {},
     exact: true,
   },
-  [SLO_CREATE_PATH]: {
+  [OLD_SLOS_WELCOME_PATH]: {
     handler: () => {
-      return <SloEditPage />;
+      return <SimpleRedirect to="/welcome" redirectToApp="slo" />;
     },
     params: {},
     exact: true,
   },
-  [SLOS_WELCOME_PATH]: {
+  [OLD_SLOS_OUTDATED_DEFINITIONS_PATH]: {
     handler: () => {
-      return <SlosWelcomePage />;
-    },
-    params: {},
-    exact: true,
-  },
-  [SLOS_OUTDATED_DEFINITIONS_PATH]: {
-    handler: () => {
-      return <SlosOutdatedDefinitions />;
-    },
-    params: {},
-    exact: true,
-  },
-  [SLO_EDIT_PATH]: {
```

Comment:
@CoenWarmer Related to my [comment above ](https://github.com/elastic/kibana/pull/177937/files#r1517391611)